l10n_it_fatturapa/models/account.py

```diff
@@ -347,66 +347,64 @@ class AccountInvoice(models.Model):
     #  2.2.2
     fatturapa_summary_ids = fields.One2many(
         'faturapa.summary.data', 'invoice_id',
-        'Fattura Elettronica Summary Data', copy=False
+        'Electronic Invoice Summary Data', copy=False
     )
     #  2.3
     vehicle_registration = fields.Date('Vehicle Registration', copy=False)
     total_travel = fields.Char('Travel in hours or Km', size=15, copy=False)
     #  2.4
     fatturapa_payments = fields.One2many(
         'fatturapa.payment.data', 'invoice_id',
-        'Fattura Elettronica Payment Data', copy=False
+        'Electronic Invoice Payment Data', copy=False
     )
     #  2.5
     fatturapa_doc_attachments = fields.One2many(
         'fatturapa.attachments', 'invoice_id',
-        'Fattura Elettronica attachments', copy=False
+        'Electronic Invoice attachments', copy=False
     )
     # 1.2.3
     efatt_stabile_organizzazione_indirizzo = fields.Char(
-        string="Indirizzo Organizzazione",
-        help="Blocco da valorizzare nei casi di cedente / prestatore non "
-             "residente, con stabile organizzazione in Italia. Indirizzo "
-             "della stabile organizzazione in Italia (nome della via, piazza "
-             "etc.)",
+        string="Organization Address",
+        help='Address of the stable organization in Italy (street name, '
```

Comment:
what about the translation of first part `Blocco da valorizzare nei casi di cedente / prestatore non residente, con stabile organizzazione in Italia`

Reply:
Forgot to add, fixed.

Thanks for the review!